dss/subscriptions_v2/__init__.py

```diff
@@ -30,10 +48,17 @@ def put_subscription(doc: dict):
 
 def get_subscription(replica: Replica, owner: str, uuid: str):
     try:
-        item = dynamodb.get_item(table=subscription_db_table.format(replica.name),
-                                 hash_key=owner,
-                                 sort_key=uuid)
-        return json.loads(item)
+        item = dynamodb.get_all_key_attributes(table=subscription_db_table.format(replica.name),
+                                               hash_key=owner,
+                                               sort_key=uuid)
+        payload = json.loads(item['body'])
+        stats = {}  # type: typing.Dict
+        for attribute_type in[SubscriptionStats.ATTEMPTS, SubscriptionStats.SUCCESSFUL, SubscriptionStats.FAILED]:
```

Comment:
Nit: you could make `SubscriptionStats` subclass `enum.Enum` and then rewrite this as

```suggestion
        for attribute_type in SubscriptionStats:
```

Reply:
wowowow resolved